docs/tutorials/services/constrain-motion.md

```diff
@@ -0,0 +1,410 @@
+---
+title: "Add Constraints and Transforms to a Motion Plan"
+linkTitle: "Add Motion Constraints"
+weight: 25
+type: "docs"
+description: "Use constraints and transforms with the Motion Service to move robot components in specific ways."
```

Comment:
```suggestion
description: "Use constraints and transforms with the Motion Service to move robot components."
```

Reply:
If we're taking that off I think I'll take off "to move robot components" since the point there was to say why you are using constraints and transforms, but you don't need constraints and transforms to move robot components in unspecific ways :)